src/uwtools/resources/jsonschema/ww3.jsonschema

```diff
@@ -0,0 +1,32 @@
+{
+  "properties": {
+    "ww3": {
+      "additionalProperties": false,
+      "properties": {
+        "namelist": {
+          "additionalProperties": false,
+          "properties": {
+            "additional_files": {
```

Comment:
We don't have a need for this with the change to the namelist task, right? In general, I think the namelist section of this driver should match the namelist section of the other drivers with `base_file` and `update_values` supported in the same way.

Reply:
OK, I'll make that change as part of my update - is there a case where a user would use `update_values`, or more for symmetry?

Reply:
Honestly, nothing wrong with keeping it as an option, so I've added it in the latest update.